src/edge_cov.rs

```diff
@@ -33,11 +33,16 @@ impl EdgeCovInspector {
         self.hitcount.fill(0);
     }
 
-    /// Get the hitcount as a byte vector.
+    /// Get an immutable reference to the hitcount.
     pub fn get_hitcount(&self) -> &[u8] {
         self.hitcount.as_slice()
     }
 
+    /// Consume the inspector and take ownership of the hitcount..
+    pub fn take_hitcount(&mut self) -> Vec<u8> {
+        std::mem::take(&mut self.hitcount)
+    }
```

Comment:
this is also problematic because this violates the non-empty constraint

```suggestion
    /// Consume the inspector and take ownership of the hitcount..
    pub fn into_hitcount(self) -> Vec<u8> {
        self.hitcount
    }
```

Reply:
I misunderstood your comment.

Btw it's fine if it's zero. We just don't want 255+1 to be represented as 0. The `edge_hash` function ensures that
https://github.com/paradigmxyz/revm-inspectors/pull/255#discussion_r1924132534